r/R/anime.R

```diff
@@ -62,6 +93,8 @@ get_matches <- function(x) {
   structure(res, class = c("tbl", "data.frame"))
 }
 
+#' @param x An `anime` object
+#' @param ... Unused, for compatibility with `print()`
```

Comment:
Same thing here let's remove everything but the export. Thank you though!

Reply:
Why export tho? Removing these causes Warnings:

With these (I agree unnecessary for practical purposes) lines:

Reply:
```
── R CMD check results ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── anime 0.0.0.9000 ────
Duration: 30.1s

❯ checking installed package size ... NOTE
    installed size is 23.7Mb
    sub-directories of 1Mb or more:
      libs  23.5Mb

0 errors ✔ | 0 warnings ✔ | 1 note ✖
```

Reply:
After removing them, as per https://github.com/JosiahParry/anime/pull/7/commits/232b2afc5e0732baf879362408b6657818e90d9f:

Reply:
Oh, no note! My bad, must have been a different one.